README.md

```
@@ -99,6 +99,111 @@ result = module.process()
+# 3. Start all services
+docker-compose up -d
+
+# 4. Check service status
+docker-compose ps
+```
+
+### Services
+
+| Service | URL | Description |
+|---------|-----|-------------|
+| Gateway (FastAPI) | http://localhost:8000 | API server with health endpoints |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)